Sakura/Modules/messages.py

```
@@ -125,47 +125,27 @@
-    "Hang on a bit, things will get better 🙃",
-    "I want to say something but can't find the words 🥺",
-    "My brain just froze 🫠",
-    "Might be a connection issue 😬",
-    "Can't really focus at the moment 😌",
-    "There's some technical glitch going on 😕",
-    "Might need a quick system reboot 🫤",
-    "I'm kinda in a confused state 😵",
-    "The API seems moody today 😤",
-    "Just a little patience, I'll be fine 💗"
+    "Ahh, that Asad always does something naughty to me. Blame him, not me. 😒",
+    "Classic Asad… always up to something naughty with me. I swear it’s never my fault! 🙄",
```

Ruff (0.13.3)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)